diff --git a/include/boost/mpi/python/skeleton_and_content.hpp b/include/boost/mpi/python/skeleton_and_content.hpp
index 2d64382..27d4da9 100644
--- a/include/boost/mpi/python/skeleton_and_content.hpp
+++ b/include/boost/mpi/python/skeleton_and_content.hpp
@@ -83,7 +83,7 @@ namespace detail {
   using boost::python::object;
   using boost::python::extract;
    
-  extern BOOST_MPI_DECL boost::python::object skeleton_proxy_base_type;
+  extern BOOST_MPI_PYTHON_DECL boost::python::object skeleton_proxy_base_type;
 
   template<typename T>
   struct skeleton_saver
